downstream/modules/platform/proc-controller-create-notification-template.adoc

```diff
@@ -7,7 +7,7 @@ Use the following procedure to create a notification template.
 .Procedure
 
 . From the navigation panel, select {MenuAEAdminJobNotifications}.
```

Comment:
I can't remember if we spoke about this, its in the section above too. The UI shows Administration > Notifiers, not Job Notifications, so we probably need to change the attribute.

Reply:
I talked to Donna about this. I thought she'd fixed it.

Reply:
I'll make the change.